src/components/editMeasure/measureDetails/measureMetadata/MeasureMetadata.tsx

```diff
@@ -82,46 +86,52 @@ export default function MeasureMetadata(props: MeasureMetadataProps) {
                 {measureMetadataType}
               </FieldLabel>
               <FieldSeparator>
-                <FieldInput
-                  type="text"
-                  name={`measure-${typeLower}`}
-                  id={`measure-${typeLower}`}
-                  autoComplete={`measure-${typeLower}`}
-                  onChange={formik.handleChange}
-                  value={formik.values.genericField}
-                  placeholder={`${measureMetadataType}`}
-                  data-testid={`measure${measureMetadataType}Input`}
-                  {...formik.getFieldProps("genericField")}
-                />
+                {canEdit && (
+                  <FieldInput
+                    type="text"
+                    name={`measure-${typeLower}`}
+                    id={`measure-${typeLower}`}
+                    autoComplete={`measure-${typeLower}`}
+                    onChange={formik.handleChange}
+                    value={formik.values.genericField}
+                    placeholder={`${measureMetadataType}`}
+                    data-testid={`measure${measureMetadataType}Input`}
+                    {...formik.getFieldProps("genericField")}
```

Comment:
an alternative to having separate fields for edit and not edit would be to just set the readonly or disabled prop equal to !canEdit

Reply:
@nmorasb That's a good idea Nate! Does that leave the input text field just not editable? I think business wanted text though, not text field.

Reply:
yeah I just went to the ticket and looked at the AC and I agree with you